pycqed/analysis_v2/Two_state_T1_analysis.py

```diff
@@ -0,0 +1,333 @@
+import lmfit
+from copy import deepcopy
+from pycqed.analysis import analysis_toolbox as a_tools
+from collections import OrderedDict
+from pycqed.analysis import measurement_analysis as ma_old
+import pycqed.analysis_v2.base_analysis as ba
+import numpy as np
+import logging
+from pycqed.analysis.tools.plotting import set_xlabel, set_ylabel
+from pycqed.analysis.tools.plotting import SI_val_to_msg_str
+
+
+class efT1_analysis(ba.BaseDataAnalysis):
+    def __init__(self, t_start: str=None, t_stop: str=None, label='',
+                 options_dict: dict=None, auto=True, close_figs=True,
+                 classification_method='rates', rates_ch_idx: int =1,
+                 ):
+        if options_dict is None:
+            options_dict = dict()
+        super().__init__(t_start=t_start, t_stop=t_stop, label=label,
+                         options_dict=options_dict, close_figs=close_figs,
+                         do_fitting=True)
+        # used to determine how to determine 2nd excited state population
+        self.classification_method = classification_method
+        self.rates_ch_idx = rates_ch_idx
+        self.d1 = 2
```

Comment:
Assuming this is the dimension? Is this attribute used in the analysis? I would guess it is not required...

Reply:
I have checked that removing 26 does not make difference.